plugins/janus_videoroom.c

```diff
@@ -593,6 +595,8 @@ int janus_videoroom_init(janus_callbacks *callback, const char *config_path) {
 				description = g_strdup(cat->name);
 			if(description == NULL) {
 				JANUS_LOG(LOG_FATAL, "Memory error!\n");
+				/* FIXME add by vivi */
+				cat = cat->next;
 				continue;
```

Comment:
`g_strdup()` will never return NULL, it will abort the program instead. So the whole if-body can be removed instead.

https://developer.gnome.org/glib/stable/glib-Memory-Allocation.html

  > If any call to allocate memory fails, the application is terminated. This also means that there is no need to check if the call succeeded.